subprojects/core/src/main/java/org/gradle/api/tasks/bundling/Zip.java

```diff
@@ -119,7 +119,7 @@ public boolean isZip64() {
      * @since 2.14
      */
     @Incubating
-    @Input @Optional
+    @Input @Nullable
```

Comment:
Nit: would be nice to have `@Nullable @Input` instead of `@Input @Nullable`.

Reply:
I changed the code so that `@Nullable` is before `@Optional`.